manager/logbroker/broker.go

```diff
@@ -301,6 +330,19 @@ func (lb *LogBroker) PublishLogs(stream api.LogBroker_PublishLogsServer) error {
 			return grpc.Errorf(codes.InvalidArgument, "missing subscription ID")
 		}
 
+		if currentSubscription == nil {
+			{
```

Comment:
Oh - never mind, bad patch. Before having a `newSubscription`, I used to do that inline in an inline func with the lock being unlocked in a defer. That got moved out, the func removed, but the extra curly braces remained